services/py-api/tests/integration_tests/test_participant_routes.py

```diff
@@ -0,0 +1,148 @@
+from typing import Any, Callable, Dict
+import pytest
+
+
+def generate_participant_request_body(
+    name: str | None = "testtest", email: str | None = "testtest@test.com", is_admin: bool | None = False, **kwargs: Any
+) -> Dict[str, Any]:
+
+    request_body = {"name": name, "email": email, "is_admin": is_admin, **kwargs}
+
+    # If any of the fields are 'None' - don't include them in the request body
+    for key in list(request_body.keys()):  # Convert to list to avoid runtime error while modifying dictionary
+        if request_body[key] is None:
+            del request_body[key]
+
+    return request_body
+
+
+@pytest.mark.asyncio
+async def test_create_random_participant(create_test_participant: Callable[..., Dict[str, Any]]) -> None:
+
+    RANDOM_PARTICIPANT_BODY = generate_participant_request_body()
```

Comment:
Since it is specific to the creation of participants and it does not have to have a cleanup phase, I did not mark it as a fixture. Put simply we would not benefit anything if we mark it as a fixture.

Reply:
Fixed in 3148caefdb44c7cce106f992f0499331ea194c75